### PR TITLE
Fix slash problem in linux and windows

### PR DESCRIPTION
**Pull request by SunweiWang**
Fix the FindBugs Parser, solve the problem the slash works differently in linux than in windows.

### CODE REVIEW

**Comment by ClintonCao** [comment]
Nice job on finally solving the problem! With this we don't have to "hard code" the path for the file.

**Comment by boninggong** [comment]
Good job Sunwei! I saw that there were some System.out.println left in your code, I'm not sure if that's for debugging or for the testing, could you look into this please?

File: src/test/java/BlueTurtle/parsers/FindBugsXMLParserTest.java [hunk 1]
```diff
@@ -56,17 +60,29 @@ public void testParseCorrectBehaviour() {
...
+	@Test
+	public void testParsingOneWarning() {
+		FindBugsXMLParser parser = new FindBugsXMLParser();
+
+		FindBugsWarning expected = new FindBugsWarning(testSet3FilePath, testSet2FileName, 47,
+				testSet2Message, testSet2Category, testSet2Priority, testSet2RuleName, testSet2Classification);
+
+		FindBugsWarning actual = (FindBugsWarning) parser.parseFile(testSet2,categoryInfo).get(0);
+		
+		System.out.println(expected.getFileName() + " |  | " + actual.getFileName());
```
Comment (boninggong) [inline src/test/java/BlueTurtle/parsers/FindBugsXMLParserTest.java]: Starting from here

File: src/test/java/BlueTurtle/parsers/FindBugsXMLParserTest.java [hunk 2]
```diff
@@ -56,17 +60,29 @@ public void testParseCorrectBehaviour() {
...
+		System.out.println(expected.getCategory() + " |  | " + actual.getCategory());
+		System.out.println(expected.getFilePath()); 
+		System.out.println(actual.getFilePath());
+		System.out.println(expected.getLine() + " |  | " + actual.getLine());
+		System.out.println(expected.getMessage() + " |  | " + actual.getMessage());
+		System.out.println(expected.getPriority() + " |  | " + actual.getPriority());
+		System.out.println(expected.getRuleName() + " |  | " + actual.getRuleName());
+		System.out.println(expected.getType() + " |  | " + actual.getType());
```
Comment (boninggong) [inline src/test/java/BlueTurtle/parsers/FindBugsXMLParserTest.java]: Till here

Comment (SunweiWang) [inline src/test/java/BlueTurtle/parsers/FindBugsXMLParserTest.java]: This is for debugging purpose, when we ran into problems with the slashes, we had to use this way to visualize it better.

**Comment by boninggong** [comment]
Thanks! Looking good now!

**Pull request merged**